website/source/docs/providers/aws/r/cloudtrail.html.markdown

```diff
@@ -16,6 +16,7 @@ resource "aws_cloudtrail" "foobar" {
     name = "tf-trail-foobar"
     s3_bucket_name = "${aws_s3_bucket.foo.id}"
     s3_key_prefix = "/prefix"
+    enable_logging = true
```

Comment:
I think this is now superfluous in the example, I'd either remove that line completely or change it to `enable_logging = false`.